### PR TITLE
ramune: add it

### PR DESCRIPTION
**Pull request by Lxkas**
add the Ramune repo into this monorepo like a BOSS

### CODE REVIEW

File: apps/ramune/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+// Check for missing environment variables
+if (missing_env_vars.length > 0) {
+	container.logger.fatal(`Missing environment variables: ${missing_env_vars.join(", ")}`);
+	process.exit(1);
+}
+
+// Register the guilds where the commands will be registered.
+// TODO: make this only in dev mode
+ApplicationCommandRegistries.setDefaultGuildIds(["1132282965368520816"]);
```
Comment (chinathaip) [inline apps/ramune/src/index.ts:34]: Which id is this again?

File: apps/ramune/src/lib/kafka.ts [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+		id: string;
+	};
+	created_at: string;
+	action: string;
+}
+
+const kafka = new Kafka({
+	clientId: "ramune-kafka-reader",
+	// TODO: broker:9092 should be based on an environment variable if running in docker
+	brokers: ["localhost:9092"]
```
Comment (chinathaip) [inline apps/ramune/src/lib/kafka.ts:28]: :3

File: apps/ramune/src/lib/kafka.ts [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+				const reviewFromKafka = {
+					reviewId: reviewMessageFromKafka.id.toString(),
+					courseCode: reviewMessageFromKafka.course.code,
+					rating: reviewMessageFromKafka.rating,
+					reviewDescription: reviewMessageFromKafka.description,
+					submittedDate: new Date(reviewMessageFromKafka.created_at)
+				};
+
+				// TODO: extract each action to its own function
```
Comment (chinathaip) [inline apps/ramune/src/lib/kafka.ts:64]: when when  >.> 5555555555555

File: apps/ramune/src/lib/review-utils.ts [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+	// body - approve
+	// {
+	//   "id":  1,
+	//   "status": "APPROVED"
+	// }
+
+	// REVIEWS_API_ADMIN_USERNAME=USERNAME
+	// REVIEWS_API_ADMIN_PASSWORD=PASSWORD
+
+	// TODO: dynamic host based on env, reviews-api:8003
```
Comment (chinathaip) [inline apps/ramune/src/lib/review-utils.ts:120]: XD

File: apps/ramune/src/.env.example [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# Bot Data
+DISCORD_TOKEN="BOT_TOKEN_HERE"
+APP_ID="APPLICATION_ID_HERE"
+
+# Config
+# NODE_ENV="production"
+DEV_GUILD_ID="DEV_GUILD_ID_HERE"
+REVIEW_CHANNEL_ID="REVIEW_CHANNEL_ID_HERE"
```
Comment (chinathaip) [inline apps/ramune/src/.env.example:8]: Also the admin and password for reviews-api admin endpoint

File: apps/ramune/src/lib/review-utils.ts [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+	// {
+	//   "id":  1,
+	//   "status": "REJECTED"
+	//   "rejectedReason": "Rejected for being a banana"
+	// }
+
+	// REVIEWS_API_ADMIN_USERNAME=USERNAME
+	// REVIEWS_API_ADMIN_PASSWORD=PASSWORD
+
+	// TODO: dynamic host based on env, reviews-api:8003
```
Comment (chinathaip) [inline apps/ramune/src/lib/review-utils.ts:158]: here too na

**Review by NayHtetKyaw — COMMENTED** [review]
image


Very accurate

File: apps/ramune/src/lib/kafka.ts (hunk 3 above)
Comment (Lxkas) [reply apps/ramune/src/lib/kafka.ts:64]: Soon:TM:
image

**Review by chinathaip — APPROVED** [review]

**Pull request merged**